diff -uNr mbank-cli-20120215/mbank-cli mbank-cli-20120215.new/mbank-cli
--- mbank-cli-20120215/mbank-cli	2012-02-15 22:48:39.000000000 +0100
+++ mbank-cli-20120215.new/mbank-cli	2012-02-23 14:36:01.943098229 +0100
@@ -70,8 +70,8 @@
 
 my $mbank_host = undef; # use set_country() to define
 my $mbank = undef; # use set_country() to define
-my $cookie_jar_file = './cookie-jar.txt';
-my $config_file = './mbank-cli.conf';
+my $cookie_jar_file = "$ENV{ HOME }/.mbank-cli_cookie-jar.txt";
+my $config_file = "$ENV{ HOME }/.mbank-cli.conf";
 my $http_read_size_hint = 1 << 20; # 1 MiB
 
 sub set_country($)
@@ -302,7 +302,7 @@
     eval { $digest_module = Digest->new('SHA-1'); } if $@;
     $digest_module = Digest->new('MD5') if $@;
   }
-  user_error "Can't open the config file: $!" unless open my $config, '<', $config_file;
+  user_error "Can't open the config file ($config_file): $!" unless open my $config, '<', $config_file;
   my $prev_digest = '';
   $main::digest_module->new();
   my $header = '';
