diff -urN mbank-cli-1.6.2.orig/mbank-cli mbank-cli-1.6.2/mbank-cli
--- mbank-cli-1.6.2.orig/mbank-cli	2017-02-06 23:53:16.304962637 +0100
+++ mbank-cli-1.6.2/mbank-cli	2017-02-06 23:53:42.481628527 +0100
@@ -556,36 +556,7 @@
 
 sub get_default_ca_path
 {
-    my ($name, @hashes) = @_;
-    my $filename = "$name.crt";
-    $filename =~ y/ /_/;
-    my $path = "/usr/share/ca-certificates/mozilla/$filename";
-    if (-r $path) {
-        return $path;
-    }
-    my $ssl_cert_dir = $ENV{SSL_CERT_DIR};
-    if (not defined($ssl_cert_dir) and $Net::SSLeay::VERSION >= 1.43) {
-        # https://www.openssl.org/docs/man1.0.2/crypto/SSLeay_version.html
-        # SSLEAY_DIR == 5
-        my $openssl_info = Net::SSLeay::SSLeay_version(5);
-        my ($openssl_dir) = $openssl_info =~ m/\AOPENSSLDIR: "(.*)"\Z/;
-        if (defined($openssl_dir)) {
-            $ssl_cert_dir = "$openssl_dir/certs";
-        }
-    }
-    if (defined($ssl_cert_dir)) {
-        for my $hash (@hashes) {
-            $path = "$ssl_cert_dir/$hash.0";
-            # TODO: actually check if this is the certificate we want
-            # hash collisions are unlikely, but not impossible
-            -r $path or next;
-            my $rpath = readlink($path) or next;
-            $rpath =~ m{\A/}
-                or $rpath = "$ssl_cert_dir/$rpath";
-            return $rpath;
-        }
-    }
-    return "$FindBin::Bin/ca.crt";
+    return '/etc/certs/ca-certificates.crt';
 }
 
 # ===========================
