--- mbank-cli-2.2.2/mbank-cli.orig	2020-02-20 16:57:45.267124994 +0100
+++ mbank-cli-2.2.2/mbank-cli	2020-02-20 16:59:43.183078639 +0100
@@ -774,30 +774,7 @@
 
 sub get_default_ca_path
 {
-    my ($name, @hashes) = @_;
-    my $filename = "$name.crt";
-    $filename =~ y/ /_/;
-    my $path = "/usr/share/ca-certificates/mozilla/$filename";
-    if (-r $path) {
-        return $path;
-    }
-    my $ssl_cert_dir = $ENV{SSL_CERT_DIR};
-    if (defined($openssl_dir)) {
-        $ssl_cert_dir //= "$openssl_dir/certs";
-    }
-    if (defined($ssl_cert_dir)) {
-        for my $hash (@hashes) {
-            $path = "$ssl_cert_dir/$hash.0";
-            # TODO: actually check if this is the certificate we want
-            # hash collisions are unlikely, but not impossible
-            -r $path or next;
-            my $rpath = readlink($path) or next;
-            $rpath =~ m{\A/}
-                or $rpath = "$ssl_cert_dir/$rpath";
-            return $rpath;
-        }
-    }
-    return "$FindBin::Bin/ca.crt";
+    return '/etc/certs/ca-certificates.crt';
 }
 
 # ===========================
