diff -uNr mbank-cli-20120215/mbank-cli mbank-cli-20120215.new/mbank-cli
--- mbank-cli-20120215/mbank-cli	2012-02-15 22:48:39.000000000 +0100
+++ mbank-cli-20120215.new/mbank-cli	2012-02-23 14:40:56.453431641 +0100
@@ -189,12 +189,12 @@
 
 sub lwp_init()
 {
-  my $ca_dir = $ENV{HTTPS_CA_DIR};
-  $ca_dir //= '/etc/ssl/certs/';
+  my $ca_file = $ENV{HTTPS_CA_FILE};
+  $ca_file //= '/etc/certs/ca-certificates.crt';
   map { delete $ENV{$_}; } grep(/^HTTPS_/, keys %ENV);
   $ENV{'HTTPS_VERSION'} = 3;
   $ENV{'HTTPS_DEBUG'} = 0;
-  $ENV{'HTTPS_CA_DIR'} = $ca_dir;
+  $ENV{'HTTPS_CA_FILE'} = $ca_file;
   umask(umask() | 077);
   my $ua = new LWP::UserAgent(
     agent => 'Mozilla/5.0',
